ci/Jenkinsfiles/build.groovy

```diff
@@ -362,14 +375,33 @@ pipeline {
       }
     }
   }
+
   post {
     always {
       script {
-        if (BRANCH_NAME == REFERENCE_BRANCH) {
+        if (!isPullRequest()) {
           // update JIRA issue
           step([$class: 'JiraIssueUpdater', issueSelector: [$class: 'DefaultIssueSelector'], scm: scm])
         }
       }
     }
+    success {
+      script {
+        if (!isPullRequest()) {
+          def prevStatus = currentBuild.getPreviousBuild()?.getResult()
+          if (!hudson.model.Result.SUCCESS.toString().equals(prevStatus) && !hudson.model.Result.UNSTABLE.toString().equals(prevStatus)) {
+            slackSend(channel: "${SLACK_CHANNEL}", color: "good", message: "Successfully built <nuxeo-explorer ${BRANCH_NAME} #${BUILD_NUMBER}|${BUILD_URL}>")
```

Comment:
`SLACK_CHANNEL` doesn't seems to be declared as an env var, could be `explorer-notifs` for instance.

Reply:
Indeed i missed the declaration. This is a platform repository, so i'll use `platform-notifs` to avoid multiplying channels.

Reply:
I do not agree. 
The goal of the `platform-notifs` Slack channel is to get notified of build failures of jobs maintained by the Platform team so that we can quickly handle the failures. If we put everything "platform" in it, there will be too much noise for us.

Reply:
Let's discuss it elsewhere else.

Reply:
Waiting for this discussion, i will remove this commit from the PR (that references another jira issue anyway)

Reply:
see #49